### ISSUE TITLE
PointConv not found

### ISSUE THREAD

**Issue by sebastianfernandezgarcia** [dragonbook/pointnet2-pytorch#8]
I am trying this implementation. When I run main.py i get this.

![image](https://user-images.githubusercontent.com/79119404/228804306-35136825-4d48-49f1-a433-3bfd63d8ef82.png)

Basically: 
ImportError: cannot import name 'PointConv' from 'torch_geometric.nn' 

When I look at torch nn documentation ( https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html )

I found: 

[PointGNNConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.PointGNNConv.html#torch_geometric.nn.conv.PointGNNConv)

[PointTransformerConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.PointTransformerConv.html#torch_geometric.nn.conv.PointTransformerConv)

[PointNetConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.PointNetConv.html#torch_geometric.nn.conv.PointNetConv)

I have tried the 3 a none seems to work. And PointConv alone i cant find it..

Do you have any Idea? 
Thanks

**Comment by sebastianfernandezgarcia** [comment]
> I am trying this implementation. When I run main.py i get this.
> 
> ![image](https://user-images.githubusercontent.com/79119404/228804306-35136825-4d48-49f1-a433-3bfd63d8ef82.png)
> 
> Basically: ImportError: cannot import name 'PointConv' from 'torch_geometric.nn'
> 
> When I look at torch nn documentation ( https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html )
> 
> I found:
> 
> [PointGNNConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.PointGNNConv.html#torch_geometric.nn.conv.PointGNNConv)
> 
> [PointTransformerConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.PointTransformerConv.html#torch_geometric.nn.conv.PointTransformerConv)
> 
> [PointNetConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.PointNetConv.html#torch_geometric.nn.conv.PointNetConv)
> 
> I have tried the 3 a none seems to work. And PointConv alone i cant find it..
> 
> Do you have any Idea? Thanks

I have found that PointNetConv is the equivalent for newer versions. They are the same method.

![image](https://user-images.githubusercontent.com/79119404/228806289-a6d9937f-2f5c-4bae-a416-c218dd1837dc.png)


How ever it still doesnt work :(

I am getting this error:

![image](https://user-images.githubusercontent.com/79119404/228807259-8bca7ec4-db4c-4dad-852d-d62df90d34b2.png)


Seems to be that RandomTranslate is deprecated, so I am using RandomJitter now.
Reading the docs, they are the same, so should work...

But now I am getting this:

![image](https://user-images.githubusercontent.com/79119404/228810030-6bd48b11-5ab9-421b-bc06-e2acd9c850d8.png)


Update: this happened becuase the implementation of this github was asuming the parameters to shapenet and in newer versions of Geometrics the datsets changed.

You have just to rename and pass the correct values.

![image](https://user-images.githubusercontent.com/79119404/228825688-c582526a-8a43-4156-8ff2-708af7894573.png)
This works for newer versions of pytorch geomtric

Then I found another problem and is related to PointConvFP and the new way of python passing super attributes,
basically:
![image](https://user-images.githubusercontent.com/79119404/228826034-611479e2-9433-4ca1-8ddf-d83d07189d2d.png)

Before there werent aggr= neither flow=, with this it´s solved

Thanks Cris for helping on this ;)
